lumispy/signals/common_luminescence.py

```diff
@@ -23,47 +23,18 @@
 
 from numpy import isnan
 from warnings import warn
+from lumispy.utils.signals import crop_edges
 
 
 class CommonLumi:
     """**General luminescence signal class (dimensionless)**"""
 
     def crop_edges(self, crop_px):
```

Comment:
To make our work for the 1.0 release easier, add a comment in the line before:
`# Deprecated, to be removed for v1.0 release`